stubs/click-default-group/click_default_group.pyi

```diff
@@ -1,30 +1,86 @@
+        context_settings: MutableMapping[str, Any] | None = None,
+        callback: Callable[..., Any] | None = None,
+        params: list[click.Parameter] | None = None,
+        help: str | None = None,
+        epilog: str | None = None,
+        short_help: str | None = None,
+        options_metavar: str | None = "[OPTIONS]",
+        add_help_option: bool = True,
+        hidden: bool = False,
+        deprecated: bool = False,
+    ) -> None: ...
     def set_default_command(self, command: click.Command) -> None: ...
     def parse_args(self, ctx: click.Context, args: list[str]) -> list[str]: ...
     def get_command(self, ctx: click.Context, cmd_name: str) -> click.Command | None: ...
     def resolve_command(self, ctx: click.Context, args: list[str]) -> tuple[str | None, click.Command | None, list[str]]: ...
     def format_commands(self, ctx: click.Context, formatter: click.HelpFormatter) -> None: ...
-    def command(self, *args, **kwargs) -> click.Command: ...  # incomplete
+    @overload
+    def command(
+        self,
+        __func: Callable[..., Any],
+        /,
+        *,
+        name: str | None = ...,
+        cls: type[click.Command] | None = ...,
+        default: Literal[False] = False,
+    ) -> click.Command: ...
+    @overload
+    @deprecated("Use default param of `DefaultGroup` or `set_default_command()` instead")
+    def command(
+        self,
+        __func: Callable[..., Any],
+        /,
+        *,
+        name: str | None = ...,
+        cls: type[click.Command] | None = ...,
+        default: Literal[True],
+    ) -> click.Command: ...
+    @overload
+    def command(
+        self, *, name: str | None = ..., cls: type[click.Command] | None = ..., default: Literal[False] = False
+    ) -> Callable[[Callable[..., Any]], click.Command]: ...
+    @overload
+    @deprecated("Use default param of `DefaultGroup` or `set_default_command()` instead")
+    def command(
+        self, *, name: str | None = ..., cls: type[click.Command] | None = ..., default: Literal[True]
+    ) -> Callable[[Callable[..., Any]], click.Command]: ...
+    @overload
+    def command(self, *args: Any, **kwargs: Any) -> Callable[[Callable[..., Any]], click.Command] | click.Command: ...
 
 class DefaultCommandFormatter:
     group: click.Group
     formatter: click.HelpFormatter
     mark: str
-    def __init__(self, group: click.Group, formatter: click.HelpFormatter, mark: str = ...) -> None: ...
+    def __init__(self, group: click.Group, formatter: click.HelpFormatter, mark: str = "*") -> None: ...
     def write_dl(self, rows: Sequence[tuple[str, str]], col_max: int = 30, col_spacing: int = -2) -> None: ...
-    def __getattr__(self, attr: str) -> Incomplete: ...
+    def __getattr__(self, attr: str) -> Any: ...  # return type depends on click.HelpFormatter attribute
     # __getattr__ used to ala-derive from click.HelpFormatter:
     # indent_increment: int
     # width: int | None
```

Comment:
This can be removed now, I think. (I have no idea what "ala-derive" means anyway.)